workflow/common/util.go

```diff
@@ -701,3 +701,24 @@ func ConvertToWorkflow(cronWf *wfv1.CronWorkflow) (*wfv1.Workflow, error) {
 	wf.SetOwnerReferences(append(wf.GetOwnerReferences(), *metav1.NewControllerRef(cronWf, wfv1.SchemeGroupVersion.WithKind(workflow.CronWorkflowKind))))
 	return wf, nil
 }
+
+func ConvertWorkflowTemplateToWorkflow(wfTemplate *wfv1.WorkflowTemplate, entrypoint string) (*wfv1.Workflow, error) {
```

Comment:
A file name "util" in a package called "common" - smell on smell.

can I suggest we put this file into another page

Reply:
Sure, where would you suggest?

Reply:
workflow/common/convert.go?